src/test/regress/sql/multi_utilities.sql

```diff
@@ -209,6 +209,10 @@ SET citus.enable_ddl_propagation to false;
 VACUUM dustbunnies;
 SET citus.enable_ddl_propagation to DEFAULT;
 
+-- test worker_hash
+SELECT worker_hash(123);
+SELECT worker_hash('1997-08-08'::date);
+
```

Comment:
Added one on composite types, I will also add a similar test to tenant isolation tests.